ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -311,7 +337,8 @@ Verify Model Inference With Retries
     ...                endpoint exposed.
     ...                This is a temporary mitigation meanwhile we find a better way to check the model
     [Arguments]    ${model_name}    ${inference_input}    ${expected_inference_output}
-    ...            ${token_auth}=${FALSE}    ${project_title}=${NONE}    ${retries}=${5}
+    ...            ${token_auth}=${FALSE}    ${project_title}=${NONE}     ${deployment_mode}=UI  ${kserve_mode}=Serverless
+    ...            ${service_port}=${NONE}   ${end_point}=${NONE}    ${retries}=${5}
```

Comment:
You are changing the order of the arguments, this will break existing tests calling this keyword expecting the 6th argument to be `retries`. Preserve the original order by adding your new arguments after the already existing ones

Reply:
ack moved it